backend/de.metas.manufacturing/src/main/java/org/eevolution/model/MPPOrder.java

```diff
@@ -442,10 +442,11 @@ public boolean reActivateIt()
 		final PPOrderId orderId = PPOrderId.ofRepoId(getPP_Order_ID());
 		orderBOMsRepo.markBOMLinesAsNotProcessed(orderId);
 
-		ModelValidationEngine.get().fireDocValidate(this, ModelValidator.TIMING_AFTER_REACTIVATE);
-
 		setDocAction(IDocument.ACTION_Complete);
 		setProcessed(false);
+
+		ModelValidationEngine.get().fireDocValidate(this, ModelValidator.TIMING_AFTER_REACTIVATE);
```

Comment:
👍 checked with MOrder and MInvoice..

Reply:
Regarding Daniela's observation for #13663, I extended the first scenario in `src/test/resources/de/metas/cucumber/features/materialCockpit/materialCockpit_SO_noProductPlanning.feature`, test case with `@Id:S0189_100`